java/core/src/main/java/com/google/protobuf/AbstractMessage.java

```diff
@@ -328,8 +328,11 @@ UninitializedMessageException newUninitializedMessageException() {
       extends AbstractMessageLite.Builder
       implements Message.Builder {
     // The compiler produces an error if this is not declared explicitly.
+    /** Method isn't abstact to bypass Java 1.6 compiler issue. */
```

Comment:
Use "// " style comment to be consistent with the line above. Also put a link to the Java issue here?